azuredevops/internal/service/core/resource_team.go

```diff
@@ -563,7 +564,11 @@ func setTeamAdministratorsPermissions(d *schema.ResourceData, clients *client.Ag
 				PrincipalPermission: securityhelper.PrincipalPermission{
 					SubjectDescriptor: item.(string),
 					Permissions: map[securityhelper.ActionName]securityhelper.PermissionType{
+						"Read":             permission,
+						"Write":            permission,
+						"Delete":           permission,
 						"ManageMembership": permission,
+						"CreateScope":      permission,
```

Comment:
Checked  the default team admin and it have 6 permissions:
```
0 = Read -> 
1 = Write -> 
2 = Delete -> 
3 = ManageMembership -> 
4 = CreateScope -> 
5 = RestoreScope -> 
```

Reply:
how exactly did you test this? I can confirm, that the `RestoreScope` bit exists, this command

`az devops security permission namespace list --organization "https://dev.azure.com/...."` gives me this result:

```
...
[
      {
        "bit": 1,
        "displayName": "View identity information",
        "name": "Read",
        "namespaceId": "5a27515b-ccd7-42c9-84f1-54c998f03866"
      },
      {
        "bit": 2,
        "displayName": "Edit identity information",
        "name": "Write",
        "namespaceId": "5a27515b-ccd7-42c9-84f1-54c998f03866"
      },
      {
        "bit": 4,
        "displayName": "Delete identity information",
        "name": "Delete",
        "namespaceId": "5a27515b-ccd7-42c9-84f1-54c998f03866"
      },
      {
        "bit": 8,
        "displayName": "Manage group membership",
        "name": "ManageMembership",
        "namespaceId": "5a27515b-ccd7-42c9-84f1-54c998f03866"
      },
      {
        "bit": 16,
        "displayName": "Create identity scopes",
        "name": "CreateScope",
        "namespaceId": "5a27515b-ccd7-42c9-84f1-54c998f03866"
      },
      {
        "bit": 32,
        "displayName": "Restore identity scopes",
        "name": "RestoreScope",
        "namespaceId": "5a27515b-ccd7-42c9-84f1-54c998f03866"
      }
    ]
...
```

but, when I create a new Project and check the permissions using this command (PowerShell):

`$namespacePermissions = Invoke-RestMethod -Uri "$organizationUri/_apis/accesscontrollists/$($namespace.namespaceId)?api-version=7.1-preview.1" -Method GET -Headers $authenticationHeader`

Then I retrieve the value `allow = 31` for the default Team administrator, which seems to be the value stored in AZDO DB.

Reply:
You are right, the admin allow value is 31.

Reply:
Lint check failed:
```log
azuredevops/internal/service/core/resource_team.go:498:44: unnecessary leading newline (whitespace)
```

Reply:
sorry, I fixed the lines